MediaBrowser.Common/Updates/IInstallationManager.cs

```diff
@@ -41,14 +41,14 @@ public interface IInstallationManager : IDisposable
         /// </summary>
         /// <param name="availablePackages">The available packages.</param>
         /// <param name="name">The name of the plugin.</param>
-        /// <param name="guid">The id of the plugin.</param>
+        /// <param name="id">The id of the plugin.</param>
         /// <param name="specificVersion">The version of the plugin.</param>
         /// <returns>All plugins matching the requirements.</returns>
         IEnumerable<PackageInfo> FilterPackages(
             IEnumerable<PackageInfo> availablePackages,
             string? name = null,
 #pragma warning disable CA1720 // Identifier contains type name
-            Guid? guid = default,
+            Guid? id = default,
 #pragma warning restore CA1720 // Identifier contains type name
```

Comment:
cant you remove the pragmas now? (since its now `id` and no longer `guid`)

Reply:
Got sniped by [`4cc19be` (#4709)](https://github.com/jellyfin/jellyfin/pull/4709/commits/4cc19be173baab9bf93a310db6790d9c33e8ad17)

Reply:
ha u fixed it while I wrote that :sweat_smile: